build-aux/snap/local/is-exec.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/python3
```

Comment:
Commenting on the first line but the comment is really about the file name. Is the intent to check if something is a dynamically linked executable? If so can you give it a somewhat more descriptive name? `is-{static,dynamic}` perhaps?

Reply:
Renamed.